config/settings/base.py

```diff
@@ -230,7 +230,6 @@ def nl_separated_bytes_list(val: str) -> List[bytes]:
     str(APPS_DIR.path("static")),
     str(ROOT_DIR.path("dist")),
```

Comment:
Will we still have static files in a `dist` directory? I thought webpack was what compiled and output things there?